src/styles/blocks/days.scss

```diff
@@ -0,0 +1,27 @@
+.day {
+  width: 100px;
+  height: 100px;
```

Comment:
The width and height of the day elements are set to fixed values of 100px. This might not be responsive on smaller screens. Consider using relative units or media queries to make the layout more flexible.